geode-assembly/src/distributedTest/java/org/apache/geode/rest/internal/web/controllers/RestAPIsOnGroupsFunctionExecutionDUnitTest.java

```diff
@@ -125,18 +123,19 @@ public void testBasicP2PFunctionSelectedGroup() {
     restURLs.clear();
   }
 
-  private class OnGroupsFunction extends RestFunctionTemplate {
+  @SuppressWarnings("unchecked")
+  private static class OnGroupsFunction extends RestFunctionTemplate {
     static final String Id = "OnGroupsFunction";
 
     @Override
     public void execute(FunctionContext context) {
-      LogWriterUtils.getLogWriter().fine("SWAP:1:executing OnGroupsFunction:" + invocationCount);
       InternalDistributedSystem ds = InternalDistributedSystem.getConnectedInstance();
       invocationCount++;
       ArrayList<String> l = (ArrayList<String>) context.getArguments();
       if (l != null) {
-        assertFalse(Collections.disjoint(l, ds.getDistributedMember().getGroups()));
+        assertThat(Collections.disjoint(l, ds.getDistributedMember().getGroups())).isFalse();
```

Comment:
I am unable to see; the changes in this PR having any relation to attributes factory removal. 
Anyways the change looks good.

Reply:
@agingade : thanks for reviewing and approving this!.
Long story short: there was a single `PR` with 2 commits: one removing usages of `AttributesFactory` and the other one fixing warnings, removing usages of other deprecated classes and replacing `junit.Assert` by `assert4j`.
Due to issues while running `StressNewTest` with 4 test files changed, I had to split the single PR in 4 (one for each test class), that's the only reason why the name doesn't match, I'll make sure to put something more meaningful while doing the squash and merge.